.github/workflows/build-yocto.yml

```diff
@@ -7,18 +7,50 @@ on:
       - main
 
 jobs:
+  kas-lock:
+    runs-on: [self-hosted, x86]
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Run kas lock
+        run: |
+          kas dump --update --lock --inplace ci/base.yml
+
+      - name: Prepare artifact
+        run: |
+          mkdir ci/lock
```

Comment:
why is that needed? Don't you just need ci/base.lock.yml?

Reply:
this needs to be rebased but I will wait for the merge of the https://github.com/quic-yocto/meta-qcom-hwe/pull/36